src/main/java/com/microsoft/azure/functions/annotation/ServiceBusQueueTrigger.java

```diff
@@ -92,4 +92,12 @@
    * @return The value indicating whether the sessions are enabled.
    */
   boolean isSessionsEnabled() default false;
+
+  /**
+   * Cardinality of the trigger input.
+   * Choose 'One' if the input is a single message or 'Many' if the input is an array of messages.
+   * 'ONE' is the default if unspecified
+   * @return An {@link Cardinality} value representing the Cardinality
+   */
+  Cardinality cardinality() default Cardinality.ONE;
```

Comment:
I believe `dataType()` will be required for cardinality. 
https://github.com/Azure/azure-functions-java-library/pull/116/files
https://github.com/Azure/azure-functions-java-library/pull/112

Reply:
I agree the datetype is already added before